src/packages/limitations.md

```diff
@@ -0,0 +1,75 @@
+# Limitations of packages
+
+When using or even considering third-party software delivered with CheriBSD
+packages, it is important to keep in mind the maturity of currently available
+CheriABI packages and potential conflicts between hybrid ABI and CheriABI
+packages.
+The following sections describe some of such limitations.
+
+## Maturity of CheriABI packages
+
+There are many projects that must still be ported to CHERI in order to provide
+a fully functional CheriABI package repository.
+The available CheriABI packages are also not continuously tested as part of the
+CheriBSD package building infrastructure.
+
+CheriABI packages compile, but many have CHERI-related warnings that
+have not been audited and only a limited set have been tested.
+While the CheriABI packages are more interesting as they use CHERI memory-safety
+features, you must remember that they might break in run-time and hence might
+not be suitable for critical operations.
+In such case, a corresponding hybrid ABI package might be considered instead of
+a CheriABI package.
+
+We expect the number of available and tested packages to grow with future
+CheriBSD releases but it will take quite some time until we will be able to
+provide a CheriABI package repository that would include all packages currently
+provided with the hybrid ABI package repository.
+
+We encourage everyone to report bugs related to CHERI support in the
+[CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+
+## Separate local bases for ABIs
+
+Upstream FreeBSD does not have any notion of an ABI at the package manager
```

Comment:
I've updated the paragraph.

Reply:
I've rephrased that a bit not to repeat "newer". Also, it's probably better no to say that one ABI is newer than another because it's not something documented on its own. A user does not have a way to keep track of the history of ABI versions to compare them. Hence, I'd prefer not to confuse the user and just say they can differ.